tfhe/src/core_crypto/entities/lwe_multi_bit_bootstrap_key.rs

```diff
@@ -183,6 +183,11 @@ impl<Scalar: UnsignedInteger, C: Container<Element = Scalar>> LweMultiBitBootstr
         bsk
     }
 
+    /// Return an iterator over the GGSW ciphertexts composing the key.
+    pub fn ggsw_iter(&self) -> impl DoubleEndedIterator<Item = GgswCiphertext<&'_ [C::Element]>> {
+        self.ggsw_list.iter()
+    }
+
```

Comment:
this should not be needed as we currently have a deref on the multi bit bsk

so bsk.iter() or &*bsk.iter() should do the trick